### PR TITLE
fix: right-click is the correct for pin in UI

### PR DESCRIPTION
**Pull request by danielo515**
## Description

Just changed the references of Left-Click to right click about pinning dependency graphs

## Motivation and Context

From my experience, Left-Click is to follow the dependency to a remote place (like github or the filesystem)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Documentation only change
- [ ] Refactor (non-breaking change which fixes an issue without changing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] :book:

  - My change doesn't require a documentation update, or ...
  - it _does_ and I have updated it

- [ ] :balance_scale:
  - The contribution will be subject to [The MIT license](https://github.com/sverweij/dependency-cruiser/blob/develop/LICENSE), and I'm OK with that.
  - The contribution is my own original work.
  - I am ok with the stuff in [**CONTRIBUTING.md**](https://github.com/sverweij/dependency-cruiser/blob/develop/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sverweij** [comment]
hi @danielo515 - thanks for this PR - good catch!

File: docs/dependency-cruiser-archi-graph.html [hunk 1]
```diff
@@ -102,10 +102,12 @@
       <span id="hint-text"></span>
       <ul>
         <li><b>Hover</b> - highlight</li>
-        <li><b>Left-click</b> - pin highlight</li>
+        <li><b>Right-click</b> - pin highlight</li>
         <li><b>ESC</b> - clear</li>
       </ul>
     </div>
+  </body>
+</html>
```
Comment (sverweij) [inline docs/dependency-cruiser-archi-graph.html:110]:
```suggestion
```

File: docs/assets/berry-high-level-dependencies.html [hunk 2]
```diff
@@ -102,10 +102,12 @@
       <span id="hint-text"></span>
       <ul>
         <li><b>Hover</b> - highlight</li>
-        <li><b>Left-click</b> - pin highlight</li>
+        <li><b>Right-click</b> - pin highlight</li>
         <li><b>ESC</b> - clear</li>
       </ul>
     </div>
+  </body>
+</html>
```
Comment (sverweij) [inline docs/assets/berry-high-level-dependencies.html:110]:
```suggestion
```

File: docs/assets/react-high-level-dependencies.html [hunk 3]
```diff
@@ -102,10 +102,12 @@
       <span id="hint-text"></span>
       <ul>
         <li><b>Hover</b> - highlight</li>
-        <li><b>Left-click</b> - pin highlight</li>
+        <li><b>Right-click</b> - pin highlight</li>
         <li><b>ESC</b> - clear</li>
       </ul>
     </div>
+  </body>
+</html>
```
Comment (sverweij) [inline docs/assets/react-high-level-dependencies.html:110]:
```suggestion
```

File: docs/dependency-cruiser-dir-graph.html [hunk 4]
```diff
@@ -102,10 +102,12 @@
       <span id="hint-text"></span>
       <ul>
         <li><b>Hover</b> - highlight</li>
-        <li><b>Left-click</b> - pin highlight</li>
+        <li><b>Right-click</b> - pin highlight</li>
         <li><b>ESC</b> - clear</li>
       </ul>
     </div>
+  </body>
+</html>
```
Comment (sverweij) [inline docs/dependency-cruiser-dir-graph.html:110]:
```suggestion
```

File: docs/dependency-cruiser-dependency-graph.html [hunk 5]
```diff
@@ -102,10 +102,12 @@
       <span id="hint-text"></span>
       <ul>
         <li><b>Hover</b> - highlight</li>
-        <li><b>Left-click</b> - pin highlight</li>
+        <li><b>Right-click</b> - pin highlight</li>
         <li><b>ESC</b> - clear</li>
       </ul>
     </div>
+  </body>
+</html>
```
Comment (sverweij) [inline docs/dependency-cruiser-dependency-graph.html:110]:
```suggestion
```

**Comment by sverweij** [comment]
Regarding the additional commits: I've rebased the PR to the latest _develop_ branch & resolved conflicts on the files in the `doc` & `docs`.

**Comment by danielo515** [comment]
@sverweij Thanks for taking care! And sorry for the trouble. I just tried a quick search and replace within github workspaces, based on VSCode, and indeed it probably thinks about itself that it is very smart.
Again, sorry for the inconvenience

**Comment by sverweij** [comment]
@danielo515 no worries! I'm glad you took the time to not only signal a bug, but also to submit a fix for it.

**Comment by danielo515** [comment]
Anytime 👍

**Pull request merged**